spi/build.gradle

```diff
@@ -44,3 +44,8 @@ shadowJar {
     relocate 'com.cronutils', 'com.amazon.opendistroforelasticsearch.jobscheduler.repackage.com.cronutils'
     relocate 'org.slf4j', 'com.amazon.opendistroforelasticsearch.jobscheduler.repackage.org.slf4j' // dependency of cron-utils
 }
+
+test {
+    // add "-Dtests.security.manager=false" to VM options if you want to run integ tests in IntelliJ
+    systemProperty 'tests.security.manager', 'false'
```

Comment:
This is required because we need to run `ESIntegTestCase` as part of SPI. This also exists as part of the Job-Scheduler `build.gradle`. Also I checked the [forum question](https://discuss.elastic.co/t/elasticsearch-5-1-1-simple-integration-test-hangs-running-gradle-test/70485)